lib/dma.h

```diff
@@ -312,53 +304,16 @@ dma_unmap_sg(dma_controller_t *dma, const dma_sg_t *sg,
     return;
 }
 
-static inline void *
```

Comment:
Are these no longer used in the vfio-user codebase? These are helpful in the Frodo code, so I will probably want them back when we abstract this into shared code.

Reply:
Yes I think they're removed because they're not used by vfio-user, however I don't see why we couldn't reinstate them. Can you raise an issue?

Reply:
already covered in #397